lucille-core/src/main/java/com/kmwllc/lucille/connector/cloudstorageclients/CloudStorageClient.java

```diff
@@ -0,0 +1,37 @@
+package com.kmwllc.lucille.connector.cloudstorageclients;
+
+import com.kmwllc.lucille.core.Publisher;
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+public interface CloudStorageClient {
```

Comment:
would be good to have some doc on what this interface is and the individual methods it includes

Reply:
updated in [41ef875](https://github.com/kmwtechnology/lucille/pull/205/commits/41ef8752a6d87d285d70ae2a101ea75109612c46)